### PR TITLE
Updated project documentation

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

**Comment by sergeyklay** [comment]
@jbboehr I've updated project documentation and refactored used links. Could you please take a look :)

**Comment by jbboehr** [comment]
Nice, I was just thinking the README needed a major overhaul!

File: README.md [hunk 1]
```diff
@@ -1,49 +1,107 @@
...
-- [PSR-6](http://www.php-fig.org/psr/psr-6/) - [psr/cache](https://github.com/php-fig/cache) - tested against [Stash](https://github.com/tedious/Stash) and [psx-cache](https://github.com/apioo/psx-cache)
-- [PSR-7](http://www.php-fig.org/psr/psr-7/) - [psr/http-message](https://github.com/php-fig/http-message) - tested against [guzzle/psr7](https://github.com/guzzle/psr7)
-- [PSR-11](http://www.php-fig.org/psr/psr-11/) - [psr/container](https://github.com/php-fig/container) - tested against [league/container](https://github.com/thephpleague/container)
-- [PSR-13](http://www.php-fig.org/psr/psr-13/) - [psr/link](https://github.com/php-fig/link) - tested against [php-fig/link-util](https://github.com/php-fig/link-util)
-- [PSR-16](http://www.php-fig.org/psr/psr-16/) - [psr/simple-cache](https://github.com/php-fig/simple-cache) - tested against [psx-cache](https://github.com/apioo/psx-cache)
+This project is aimed to provide the accepted PSR interfaces in PHP extension form.
+You can use interfaces provided by this extension easily, [exactly in the same way as you normally do][:example:].
```
Comment (jbboehr) [inline README.md:12]: 'exactly in the same way as you normally do' isn't quite right as extensions compiled into PHP can be linked to, whereas you need to go through the module system for this extension.

**Comment by jbboehr** [comment]
@sergeyklay I've made some adjustments to your changes to the README, if you'd like to review: https://github.com/jbboehr/php-psr/compare/jbboehr-patch-1?expand=1

**Comment by sergeyklay** [comment]
@jbboehr Looks good to me!

**Pull request merged**